src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -1829,6 +1838,15 @@ def taxes_and_insurance_cost_rule(c):
             return c.property_taxes_and_insurance_cost == 0.01 * TPC
 
         # sum of fixed O&M costs
+
+        # sum of fixed operating costs of membrane units
+        @b.Constraint()
+        def sum_watertap_fixed_cost(c):
+            if b.watertap_fixed_costs_list is None:
+                return c.watertap_fixed_costs == 0
+            else:
+                return c.watertap_fixed_costs == sum(b.watertap_fixed_costs_list)
```

Comment:
I took a look at the source code for the WaterTAP costing and the values in `watertap_fixed_costs_list` have default units of [currency]/year. Since we defined `c.watertap_fixed_costs` in [currency], I'd suggest:
```suggestion
                return c.watertap_fixed_costs == sum(b.watertap_fixed_costs_list) * pyunits.year
```